roachtest/allocbench: Add labels to the std-dev metric

csgourav opened the pull request:
Adds unit and is_higher_better tags to std-dev metrics being emitted from allocbench tests.
Adds code for aggregate value generation in cdc/scan test.
Adds missing unit label for tpcc test.

Epic: none
Fixes: none

csgourav committed:
roachtest/allocbench: Add labels to the std-dev metric

Adds unit and is_higher_better tags to std-dev metrics
being emitted from allocbench tests.

Epic: none
Fixes: none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/144218)
<!-- Reviewable:end -->

csgourav committed:
roachtest/cdcbench: generate aggregate stats for cdc/scan tests

Adds aggregate metric for cdc/scan tests.
Adds one missing warehouse label in tpcc test.

Epic: none
Fixes: none

sambhav-jain-16 commented:
Thanks @csgourav for the changes.
Can you run a teamcity job and post the output link here?

csgourav commented:
For allocbench tests, Teamcity run [link](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19384362?buildTab=artifacts#%2Fallocbench%2Fnodes%3D7%2Fcpu%3D8%2Fkv%2Fr%3D95%2Faccess%3Dskew%2Frun_1%2F1.perf)

Sample output of aggregated stats

`# TYPE std_cpu___ gauge
std_cpu___{branch="144218",cloud="gce",commit="8b29e48029a",goarch="amd64",goos="linux",owner="kv",suite="nightly",test="allocbench/nodes=7/cpu=8/kv/r=95/access=skew",test_run_id="teamcity-19384362",unit="stddev",is_higher_better="false"} 0.936965 1744280247

'# TYPE cpu___ gauge
cpu___{branch="144218",cloud="gce",commit="8b29e48029a",goarch="amd64",goos="linux",owner="kv",suite="nightly",test="allocbench/nodes=7/cpu=8/kv/r=95/access=skew",test_run_id="teamcity-19384362",unit="percent",is_higher_better="false"} 8.790000 1744280247

'# TYPE write___ gauge
write___{branch="144218",cloud="gce",commit="8b29e48029a",goarch="amd64",goos="linux",owner="kv",suite="nightly",test="allocbench/nodes=7/cpu=8/kv/r=95/access=skew",test_run_id="teamcity-19384362",unit="percent",is_higher_better="false"} 2.430000 1744280247

'# TYPE cost_gb_ gauge
cost_gb_{branch="144218",cloud="gce",commit="8b29e48029a",goarch="amd64",goos="linux",owner="kv",suite="nightly",test="allocbench/nodes=7/cpu=8/kv/r=95/access=skew",test_run_id="teamcity-19384362",unit="GB",is_higher_better="false"} 0.000000 1744280247

'# TYPE std_write___ gauge
std_write___{branch="144218",cloud="gce",commit="8b29e48029a",goarch="amd64",goos="linux",owner="kv",suite="nightly",test="allocbench/nodes=7/cpu=8/kv/r=95/access=skew",test_run_id="teamcity-19384362",unit="stddev",is_higher_better="false"} 0.427177 1744280247

'# TYPE std_cost_gb_ gauge
std_cost_gb_{branch="144218",cloud="gce",commit="8b29e48029a",goarch="amd64",goos="linux",owner="kv",suite="nightly",test="allocbench/nodes=7/cpu=8/kv/r=95/access=skew",test_run_id="teamcity-19384362",unit="stddev",is_higher_better="false"} 0.000000 1744280247`

csgourav commented:
TFTR
bors r=sambhav-jain-16

csgourav commented:
bors r=sambhav-jain-16
